fdbbackup/backup.actor.cpp

```diff
@@ -697,6 +697,9 @@ static void printAgentUsage(bool devhelp) {
 		   "  --logdir PATH  Specifes the output directory for trace files. If\n"
 		   "                 unspecified, defaults to the current directory. Has\n"
 		   "                 no effect unless --log is specified.\n");
+	printf("  --trace_format FORMAT\n"
+	       "                 Select the format of the trace files. xml and json supported.\n"
```

Comment:
Can you also document that the default is xml?

Reply:
Done